--- /etc/s6-overlay/s6-rc.d/syslogd-log/run	2022-08-26 17:57:50.000000000 +0200
+++ /etc/s6-overlay/s6-rc.d/syslogd-log/run.new	2022-08-28 20:23:36.637010356 +0200
@@ -2,12 +2,15 @@
 
 s6-envuidgid -D 32761:32761: -- sysllog
 s6-applyuidgid -U --
-backtick -D "n20 s1000000" line { printcontenv S6_LOGGING_SCRIPT }
+backtick -D "n5 s1000000" line { printcontenv S6_LOGGING_SCRIPT }
+backtick -D "false" -E fileLOG { printcontenv LOG2FILE }
 multisubstitute
 {
   importas -usC T line
   define dir /var/log/syslogd
 }
+backtick -D "1" logdir_vdr { if { test $fileLOG = true } echo "1" "$T" "/vdr/log" }
+importas -usC LOGDIR_VDR logdir_vdr
 exec -c
 s6-log -d3 --
 
@@ -20,4 +23,5 @@
  - +mail\\.  $T ${dir}/mail
  - +user\\.  $T ${dir}/user
  - +^[[:alnum:]]*\\.info: +^[[:alnum:]]*\\.notice: +^[[:alnum:]]*\\.warn: -^auth\\. -^authpriv\\. -^cron\\. -daemon\\. -mail\\.  $T ${dir}/messages
+ - +^[[:alnum:]].*[[:space:]]vdr: +^[[:alnum:]].*[[:space:]]recordingaction: +^[[:alnum:]].*[[:space:]]CheckRecording: +^[[:alnum:]].*[[:space:]]NaluDumper:  ${LOGDIR_VDR}
  + -^auth\\. -^authpriv\\.  $T ${dir}/everything
