Description: Fix epg SegmentTransfer logic
Author: Jörg Wendel
Origin: https://www.vdr-portal.de/forum/index.php?thread/134876-vdr-epg2vdr-compilieren-schl%C3%A4gt-mit-vdr-2-6-fehl/&postID=1346583#post1346583

Index: vdr-2.6.1/eit.c
===================================================================
--- vdr-2.6.1.orig/eit.c	2022-02-02 15:06:04.058454694 +0100
+++ vdr-2.6.1/eit.c	2022-02-02 15:06:04.054452695 +0100
@@ -115,22 +115,23 @@
      return;
      }
 
-  if (!EpgHandlers.BeginSegmentTransfer(Channel)) {
+  cSchedule *pSchedule = (cSchedule *)Schedules->GetSchedule(Channel, true);
+
+  if (pSchedule->OnActualTp(Tid) && (Tid & 0xF0) == 0x60) {
      SchedulesStateKey.Remove(false);
      ChannelsStateKey.Remove(false);
      return;
      }
 
-  bool ChannelsModified = false;
-  bool handledExternally = EpgHandlers.HandledExternally(Channel);
-  cSchedule *pSchedule = (cSchedule *)Schedules->GetSchedule(Channel, true);
-
-  if (pSchedule->OnActualTp(Tid) && (Tid & 0xF0) == 0x60) {
+  if (!EpgHandlers.BeginSegmentTransfer(Channel)) {
      SchedulesStateKey.Remove(false);
      ChannelsStateKey.Remove(false);
      return;
      }
 
+  bool ChannelsModified = false;
+  bool handledExternally = EpgHandlers.HandledExternally(Channel);
+
   bool Empty = true;
   bool Modified = false;
   time_t LingerLimit = Now - EPG_LINGER_TIME;
